yuv_rgb.h

```diff
@@ -0,0 +1,155 @@
+// Copyright 2016 Adrien Descamps
+// Distributed under BSD 3-Clause License
```

Comment:
Would you mind adding a comment that explains where the yuv_rgb.h/cc files are copied from, so that we get some traceability for future updates of this code? You can either add the comment at the top of the files or in the commit comment.